.betterer.results

```diff
@@ -1,5 +1,5 @@
 // BETTERER RESULTS V2.
-//
+// 
```

Comment:
Is this intended?

Reply:
Nope

Reply:
I checked on [main](https://github.com/grafana/grafana/blob/d0312af571ba8460c5dab26282e0f7000730e4a5/.betterer.results#L2) and it seems line 2 is already there. Is it just the blank space?

Reply:
@JoaoSilvaGrafana fixed!